lrc/Pos/Lrc/Core.hs

```diff
@@ -27,10 +27,12 @@ import           Pos.Util.Util       (getKeys)
 -- 2. Delegates who became richmen.
 findDelegationStakes
     :: forall m . Monad m
-    => (StakeholderId -> m Bool)         -- ^ Check if user is issuer?
-    -> (StakeholderId -> m (Maybe Coin)) -- ^ Gets effective stake.
-    -> Coin
-    -> Sink (StakeholderId, HashSet StakeholderId) m (RichmenSet, RichmenStake) -- old richmen, new richmen
+    => (StakeholderId -> m Bool)                   -- ^ Check if user is issuer?
+    -> (StakeholderId -> m (Maybe Coin))           -- ^ Gets effective stake.
+    -> Coin                                        -- ^ Coin threshold
+    -> Sink (StakeholderId, HashSet StakeholderId) -- ^ Old richmen, new richmen
```

Comment:
I think comment is on the wrong line. Even if it doesn't confuse haddock, it confuses reader, because it applies to `(RichmenSet, RichmenStake)`, not to `(StakeholderId, HashSet StakeholderId)`.